bintray/utils.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
This file is a dup of https://github.com/soulcycle/ci-tools/blob/master/gcr/utils.sh so we can delete, since if we're installing bintray scripts in the CI, we'd also be installing GCR scripts